spec/futurist/forking_resolution_strategy_spec.rb

```diff
@@ -102,14 +102,12 @@
   end
 
   it "is ready after the process has exited" do
-    def ready_monitor_initializer(_process_id)
-      stub_monitor(ready: true)
-    end
-    ready_monitor_constructor = method(:ready_monitor_initializer)
     allow_any_instance_of(Futurist::ForkingResolutionStrategy).
       to receive(:start_promise_evaluation)
     strategy = Futurist::ForkingResolutionStrategy.new(
-      process_monitor_constructor: ready_monitor_constructor,
+      process_monitor_constructor: ->(_process_id) {
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Use the `lambda` method for multi-line lambdas.